src/lib/migration-chunks/validation/errors.ts

```diff
@@ -109,6 +112,52 @@ const errorCreators: ErrorCreators = {
       HAS_ENTRIES: (id) => {
         return `Content type "${id}" cannot be deleted because it has entries.`
       }
+    },
+    transformEntries: {
+      CONTENT_TYPE_DOES_NOT_EXIST: (id) => {
+        return `You cannot transform entries for content type "${id}" because it does not exist.`
+      },
+      TRANSFORM_AFTER_CONTENT_TYPE_DELETE: (id) => {
+        return `You cannot transform entries for content type "${id}" because it was deleted.`
+      },
+      TRANSFORM_BEFORE_CONTENT_TYPE_CREATE: (id) => {
+        return `You cannot transform entries for content type "${id}" because it has not yet been created.`
+      }
+    },
+    deriveEntries: {
```

Comment:
question: do we run the same validations for both the source and target CTs?

Reply:
yes!